### DISCUSSION TITLE
School Blocked my settings

### DISCUSSION THREAD

**Discussion by 0Zyther0** [3kh0/ext-remover#478] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
What should I do, I was trying to to the kill curly method but I just realized that my school blocked the settings on chrome. Is there a way to unblock it?

**Comment by Aka-but-nice** [comment]
DM me on discord

**Reply by 0Zyther0** [reply]
What's is your discord?

**Reply by 0Zyther0** [reply]
Mine is Zyther#4817

**Reply by Aka-but-nice** [reply]
ok

**Comment by ghost** [comment, marked as answer]
You can ctrl+shift+backspace. Then do esc. And done! 

Please note that if you're using a link like chrome://settings/autofill it will be blocked.

**Reply by 0Zyther0** [reply]
it doesn't work
![Screenshot 2023-02-06 2 11 06 AM](https://user-images.githubusercontent.com/124122876/216906892-631fad87-8554-457a-a47b-ee82b6052e2c.png)

**Comment by bartholemuew** [comment]
same problem
